nodes/cma/nodes-cma-autoscaling-custom-rn.adoc

```diff
@@ -30,23 +30,38 @@ The following table defines the Custom Metrics Autoscaler Operator versions for
 |{product-title} version
 |General availability
 
-|2.11.2
+|2.12.1
 |4.15
 |General availability
 
-|2.11.2
+|2.12.1
 |4.14
 |General availability
 
-|2.11.2
+|2.12.1
 |4.13
 |General availability
 
-|2.11.2
+|2.12.1
 |4.12
 |General availability
 |===
 
+[id="nodes-pods-autoscaling-custom-rn-2121-376_{context}"]
+== Custom Metrics Autoscaler Operator 2.12.1-376 release notes
+
+This release of the Custom Metrics Autoscaler Operator 2.12.1-376 provides security updates and bug fixes for running the Operator in an {product-title} cluster. The following advisory is available for the link:https://access.redhat.com/errata/RHSA-TBD[RHSA-TBD].
+
+[IMPORTANT]
+====
+Before installing this version of the Custom Metrics Autoscaler Operator, remove any previously installed Technology Preview versions or the community-supported version of KEDA.
+====
+
+[id="nodes-pods-autoscaling-custom-rn-2121-376-bugs_{context}"]
+=== Bug fixes
+
+* Previously, if invalid values, such as nonexistent namespaces, were specified in scaled object metadata, the underling scaler clients would not free/close their client descriptors, resulting in a slow memory leak. This fix properly closes the underlying client descriptors when there are errors, preventing memory from leaking. (link:https://issues.redhat.com/browse/OCPBUGS-30145[*OCPBUGS-30145*])
```

Comment:
hahaha apparently I typed "underling" the first time, sorry. As funny as that would have been... :smile: 
```suggestion
* Previously, if invalid values, such as nonexistent namespaces, were specified in scaled object metadata, the underlying scaler clients would not free/close their client descriptors, resulting in a slow memory leak. This fix properly closes the underlying client descriptors when there are errors, preventing memory from leaking. (link:https://issues.redhat.com/browse/OCPBUGS-30145[*OCPBUGS-30145*])
```

Reply:
@jkyros One of those typos that passes the spell checker! Thanks.